doc/main/intro/limitations.rst

```diff
@@ -44,3 +44,10 @@ Incorrect Installation Location
 **Limitation:** oneTBB does not support ``fork()``. 
 
 **Solution:** To work-around the issue, consider using ``task_scheduler_handle`` to join oneTBB worker threads before using ``fork()``.
+
+Custom Allocator and Topology API Incompatibilities
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+**Limitation:** On Linux* OS, using custom memory allocators together with ``tbb::info`` and ``tbb::task_arena::constraints`` APIs may result in undefined behavior.
```

Comment:
I don't like the formulation `custom memory allocators` here since it can be interpreted as usage of custom allocator types. Since this issue only relates to the malloc replacement libraries like tbbmalloc, jemalloc, etc., I propose to use "dynamic malloc replacement" or "dynamic allocating functions replacement" instead.

Reply:
Also, do we use "undefined behavior" when defining the known limitations? It seems like we can be more concrete and say it is runtime failures.